it/controllers/employment/EmploymentTaxControllerISpec.scala

```diff
@@ -0,0 +1,163 @@
+    val expectedH1: String = s"How much UK tax was taken from your clients maggie earnings?"
+    val expectedTitle: String = s"How much UK tax was taken from your client’s earnings?"
+    val expectedP1: String = "You can usually find this amount in the Pay and Income Tax details section of your clients P60."
+    val expectedP2: String = s"If £6,782.92 was not taken in UK tax, tell us the correct amount."
+
+  }
+
+  object ExpectedIndividualCY extends SpecificExpectedResults {
+    val expectedH1: String = s"How much UK tax was taken from your maggie earnings?"
+    val expectedTitle: String = s"How much UK tax was taken from your earnings?"
+    val expectedP1: String = "You can usually find this amount in the Pay and Income Tax details section of your P60."
+    val expectedP2: String = s"If £6,782.92 was not taken in UK tax, tell us the correct amount."
+
+  }
+
+  object ExpectedAgentCY extends SpecificExpectedResults {
+    val expectedH1: String = s"How much UK tax was taken from your clients maggie earnings?"
+    val expectedTitle: String = s"How much UK tax was taken from your client’s earnings?"
+    val expectedP1: String = "You can usually find this amount in the Pay and Income Tax details section of your clients P60."
+    val expectedP2: String = s"If £6,782.92 was not taken in UK tax, tell us the correct amount."
+  }
+
+  val userScenarios: Seq[UserScenario[CommonExpectedResults, SpecificExpectedResults]] = {
+    Seq(UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN)),
+      UserScenario(isWelsh = false, isAgent = true,  CommonExpectedEN, Some(ExpectedAgentEN)),
+      UserScenario(isWelsh = true, isAgent = false, CommonExpectedCY, Some(ExpectedIndividualCY)),
+      UserScenario(isWelsh = true, isAgent = true, CommonExpectedCY, Some(ExpectedAgentCY)))
+  }
+
+  object Selectors {
+    val p1 = "#main-content > div > div > form > div > label > p:nth-child(2)"
+    val p2 = "#main-content > div > div > form > div > label > p:nth-child(3)"
+    val hintText = "#amount-hint"
+    val currencyBox = "#amount"
+    val continueButton = "#continue"
+    val caption = "#main-content > div > div > form > div > label > header > p"
+  }
+
+  object Model {
+
+    val employmentSource1 = EmploymentDetails(
+      "Mishima Zaibatsu",
+      employerRef = Some("223/AB12399"),
+      startDate = Some("2019-04-21"),
+      currentDataIsHmrcHeld = true
+    )
+    val employmentId = "223/AB12399"
+    val employmentCyaModel = EmploymentCYAModel(employmentSource1)
+    val employmentUserData = EmploymentUserData(sessionId, mtditid, nino, taxYear, employmentId, false, employmentCyaModel)
+  }
+
+  ".show" when {
+    import Selectors._
+
+    userScenarios.foreach{ user =>
+      s"language is ${welshTest(user.isWelsh)} and request is from an ${agentTest(user.isAgent)}" should {
+        import Selectors._
+
+        //noinspection ScalaStyle
+        "for in year return a page with filled box" which {
```

Comment:
in year should always redirect so test case should be updated to say end of year